app/(tabs)/explore.tsx

```diff
@@ -1,9 +1,240 @@
-import { ScrollView, Text } from 'react-native';
+import { useState } from 'react';
+import { Modal, Pressable, Text, View, Platform, ScrollView, TextInput } from 'react-native';
+import DateTimePicker from '@react-native-community/datetimepicker';
+import { Card } from '@/components/ui/card';
+import ScreenWrapper from '@/components/ScreenWrapper';
+import { CarpoolCard } from '@/components/CarpoolCard';
+
+export default function RidesScreen() {
+  const [selectedDate, setSelectedDate] = useState<Date>(new Date());
+  const [showPicker, setShowPicker] = useState(false);
+  const [mode, setMode] = useState<'date' | 'time'>('date');
+  const [searchQuery, setSearchQuery] = useState('');
+
+  const handleChange = (event: any, date?: Date) => {
+    if (event.type === 'dismissed') {
+      setShowPicker(false);
+      return;
+    }
+
+    if (date) {
+      setSelectedDate(date);
+    }
+
+    if (Platform.OS === 'android') {
+      setShowPicker(false);
+    }
+  };
+
+  const showDatePicker = () => {
+    setMode('date');
+    setShowPicker(true);
+  };
+
+  const isToday = selectedDate.toDateString() === new Date().toDateString();
 
-export default function ExploreScreen() {
   return (
-    <ScrollView>
-      <Text>Explore Screen</Text>
-    </ScrollView>
+    <ScreenWrapper>
```

Comment:
de main view een scrollview is niet goed nu scrollt de container en de main view

Reply:
niet een comment resolven tot dat die ook helemaal klaar is met de discussie.

de main view moet nooit scrollen als er al een scrollview in de container zit, een double scroll is slechte ux.

wat hier moet gebeuren is dat de container altijd in view blijft en dat je alle cards can zien in de container door daar te scrollen